### DISCUSSION TITLE
dynamic mask - trigger dispatch from another form field

### DISCUSSION THREAD

**Discussion by kelenakamura** [uNmAnNeR/imaskjs#913] [Q&A]
it seems the dispatch function is triggered on input of the masked input? is there a way to trigger this onChange from another form field (ie a select box)?

would like to have a country selector change the input mask used on a text input (ie phone field).

**Comment by uNmAnNeR** [comment]
`dispatch` option is needed when mask is dynamically changed during input. in your case you need to set the mask.
